dvc/repo/__init__.py

```diff
@@ -93,7 +93,9 @@ def find_dvc_dir(root=None):
         return os.path.join(root_dir, Repo.DVC_DIR)
 
     def remind_to_git_add(self):
-        if not self.files_to_git_add:
+        is_git_repo = type(self.scm).__name__ == "Git"
```

Comment:
Should we check it by name? Maybe something like:
`type(self.scm) is dvc.scm.Git`?

Reply:
uff, great one, @pared :+1: